test/spec/runners/bls.ex

```diff
@@ -60,8 +60,18 @@ defmodule BLSTestRunner do
     end
   end
 
-  defp assert_sign(_input, _output) do
-    assert false
+  defp assert_sign(input, output) do
+    %{message: message, privkey: private_key} = input
+
+    case output do
+      nil ->
+        {result, _error_msg} = Bls.sign(private_key, message)
+        assert result == :error
+
+      output ->
+        {:ok, signature} = Bls.sign(private_key, message)
+        assert signature == output
```

Comment:
Prepending `assert` on match assignments improves error messages.
```suggestion
      nil ->
        assert {result, _error_msg} = Bls.sign(private_key, message)
        assert result == :error

      output ->
        assert {:ok, signature} = Bls.sign(private_key, message)
        assert signature == output
```